engine/graphics/src/graphics.h

```diff
@@ -337,6 +336,23 @@ namespace dmGraphics
         uint64_t m_PolygonOffsetFillEnabled : 1;
     };
 
+    const static uint8_t MAX_VERTEX_STREAM_COUNT = 8;
+
+    struct VertexStream
```

Comment:
I'd even move these to graphics_private.h, since I don't think the other engine libraries should know about the struct layout.